trieste/acquisition/function.py

```diff
@@ -392,12 +393,16 @@ def __init__(self, sample_size: int, model: ProbabilisticModel):
         tf.debugging.assert_positive(sample_size)
 
         self._sample_size = sample_size
+
+        # _eps is essentially a lazy constant, declared here but initialised only on the first call
```

Comment:
Same comment as in `BatchReparametrizationSampler` regarding use of the term "initialised".